pkg/cli/admin/nodeimage/create.go

```diff
@@ -381,7 +381,7 @@ func (o *CreateOptions) copyArtifactsFromNodeJoinerPod() error {
 	}
 	if o.GeneratePXEFiles {
 		rsyncOptions.Source.Path = "/assets/boot-artifacts/"
-		rsyncOptions.RsyncInclude = []string{"*.img", "*.*vmlinuz", "*.ipxe"}
+		rsyncOptions.RsyncInclude = []string{"*.img", "*.*vmlinuz", "*.ipxe", "*.ins", "*.addrsize"}
```

Comment:
@andfasano would it be valid to make it just download all non-invisible files from boot-artifacts?
This looks fine, but ideally there shouldn't be any need to co-ordinate changes between the installer and oc whenever the installer changes, since oc has to work with the next version of the installer.

```suggestion
		rsyncOptions.RsyncInclude = nil
		rsyncOptions.RsyncExclude = []string{".*"}
```

Reply:
(my previous comment seems to be lost)
That's something we recently fixed in https://github.com/openshift/oc/pull/1931, so copying everything may bring back unwanted stuff.